app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class InvitesController < ApplicationController
+  before_action :check_user_registration, :authenticate_user!
+  before_action :fetch_current_invite, only: [:accept_invite]
+  before_action :check_user_organization, only: %i[index create]
```

Comment:
Why are you using %i[index create] in this line and in the previous line, a regular symbols array?